pkg/skullking/deck.go

```diff
@@ -35,9 +35,13 @@ const (
 	CardTypeTigress   CardType = "Tigress"
 
 	CardTypeSuitParrot        CardType = "Parrot"
+	CardTypeSuitGreen         CardType = "green"
 	CardTypeSuitTreasureChest CardType = "Treasure Chest"
-	CardTypeSuitJollyRoger    CardType = "Jolly Roger"
+	CardTypeSuitYellow        CardType = "yellow"
 	CardTypeSuitPirateMap     CardType = "Pirate Map"
+	CardTypeSuitPurple        CardType = "purple"
+	CardTypeSuitJollyRoger    CardType = "Jolly Roger"
+	CardTypeSuitBlack         CardType = "black"

```

Comment:
I don't think colors and suit names behave the same. For example, Black and JollyRoger should be the same, but they are not.

Maybe we can consider dropping the names and leave only the colors

Reply:
I just deleted every instance of the old naming convention